src/EA.Weee.Web/Scripts/weee-application.js

```diff
@@ -171,6 +171,7 @@
             //source: suggest,
             element: newElement,
             name: element.id + "-auto",
+            displayMenu: 'overlay',
```

Comment:
is that for the drop downs UX bug, raised by Sarah?

Reply:
Yeah it is, thankfully there was an option on the control :)